saleae/saleae.py

```diff
@@ -136,6 +136,7 @@ def kill_logic(kill_all=False):
 		# conservative assumption that if there's only one process running with
 		# 'logic' in the name, that it's Saleae Logic. If kill_all is set to
 		# True, all processes with 'logic' in the name are killed.
+		# Note that is_logic_running method rely on the same trick.
```

Comment:
Since there's already one other change, if you wouldn't mind fixing a small grammar nit too: s/rely/relies